modules/apps/object/object-api/src/main/java/com/liferay/object/exception/ObjectDefinitionExternalReferenceCodeException.java

```diff
@@ -49,9 +50,9 @@ public static class MustBeLessThan75Characters
 
 		public MustBeLessThan75Characters() {
 			super(
-				Collections.singletonList(75),
+				Arrays.asList(75, "external-reference-code"),
```

Comment:
Did you guys test if the key `external-reference-code` is displayed as a word, not a key?

Reply:
@larissacribeiro @murilostodolni

Reply:
Yes, @marcelabc, I did! It's being displayed like this:

![erc_lenght_error](https://github.com/liferay-objects/liferay-portal/assets/82094673/fd18d8ec-053c-476a-b819-59f43a1b5293)

Reply:
Thanks @larissacribeiro !